receiver/purefareceiver/README.md

```diff
@@ -12,12 +12,15 @@
 [contrib]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
 <!-- end autogenerated section -->
 
-The Pure Storage FlashArray receiver, receives metrics from Pure Storage internal services hosts.
+The Pure Storage FlashArray receiver, receives metrics from the Pure Storage FlashArray.
 
 ## Configuration
 
 The following settings are required:
- -  `endpoint` (default: `http://172.0.0.0:9490/metrics/array`): The URL of the scraper selected endpoint
+ -  `endpoint` (default: `http://127.0.0.0:9490/metrics/array`): The URL of the scraper selected endpoint.
+ -  `fa_array_name` (no default): The array's pretty name to be used as a metrics label.
```

Comment:
It looks like the new "namespace" attribute is also required. It needs to be placed here.

Reply:
fixed.